spec/linter-pycodestyle-spec.js

```diff
@@ -0,0 +1,60 @@
+'use babel';
+
+import { join } from 'path';
+import { beforeEach, it } from 'jasmine-fix';
+
+const fixturePath = join(__dirname, 'fixtures');
+const goodPath = join(fixturePath, 'good.py');
+const badPath = join(fixturePath, 'bad.py');
+const emptyPath = join(fixturePath, 'empty.py');
+
+describe('The pycodestyle provider for Linter', () => {
+  const lint = require('../lib/main.coffee').provideLinter().lint;
+
+  beforeEach(async () => {
+    // Info about this beforeEach() implementation:
+    // https://github.com/AtomLinter/Meta/issues/15
+    const activationPromise = atom.packages.activatePackage('linter-pycodestyle');
+
+    await atom.packages.activatePackage('language-python');
+
+    atom.packages.triggerDeferredActivationHooks();
+    await activationPromise;
+  });
+
+  it('should be in the packages list', () =>
+    expect(atom.packages.isPackageLoaded('linter-pycodestyle')).toBe(true),
+  );
+
+  it('should be an active package', () =>
+    expect(atom.packages.isPackageActive('linter-pycodestyle')).toBe(true),
+  );
+
+  describe('checks bad.py and', () => {
+    let editor = null;
+    beforeEach(async () => {
+      editor = await atom.workspace.open(badPath);
+    });
+
+    it('finds at least one message', async () => {
```

Comment:
🔥 I've been moving specs to just drop this one and check everything in the test after this.